script/component-migration/index.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable no-console */
+const chalk = require('chalk');
+const commandLineArgs = require('command-line-args');
+const fs = require('fs');
+const glob = require('glob');
+const replacements = require('./transformers');
+
+const optionDefinitions = [
+  { name: 'component', alias: 'c' },
+  { name: 'dir', alias: 'd' },
+];
+const options = commandLineArgs(optionDefinitions);
+
+const FILENAMES = glob.sync(`${options.dir}/**/*.jsx`);
+
+const legacyImport = `import ${
+  options.component
+} from '@department-of-veterans-affairs/component-library/${
+  options.component
+}'`;
+
+function handleError(error) {
+  console.log(error);
+}
+
+function translateProps(componentString, propMap) {
+  // Make a copy so we don't modify the reference
+  let translatedComp = componentString.toString();
+
+  Object.entries(propMap)
+    .sort((pairOne, _) => {
+      const children = ['content', 'children'];
+      if (children.includes(pairOne[0])) return -1;
```

Comment:
`pairOne` refers to the first item to be comparred in the comparator function: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#parameters

In this case it's a pair because `Object.entries` gives us a pair: the key, and the value. Would `propPair` be more helpful?

Reply:
I went ahead and destrctured the pair so we can work with `prop` directly :shrug: